qcelemental/models/results.py

```diff
@@ -270,6 +270,12 @@ class AtomicResultProtocols(ProtoModel):
         WavefunctionProtocolEnum.none, description=str(WavefunctionProtocolEnum.__doc__)
     )
     stdout: bool = Field(True, description="Primary output file to keep from a Result computation")
+    error_correction_retries: int = Field(
```

Comment:
Unsure if this particularly a count issue. Imagine I need to fix three issues similar to "autosym" in NWChem, this feels more like a "fix all inconsequential errors" enum. "inconsequential" of course will be something hard to define.

This is separate from try to fix convergence through 1-3 ranks:
 - rank 1, default
 - rank 2, SOSCF
 - rank 3, SOSCF + GDIIS + level shift

Reply:
Second the "count" comment, as I was typing that ppl would likely have this at 0 or 50 because correctable errors of this tweak type are independent. So perhaps a bool would do? The 1-3 ranks Daniel describes are perhaps a `GuidableError` that are managed by separate protocols.

Reply:
To make sure I follow your suggestion correctly, is the idea to have the user specify a "level of consequentialness." Where higher levels will allow QCEngine to make more aggressive changes to an input file?

Reply:
I think so. I say *think* here because I am still working through the consequences of this. At the moment, I am working through custodian pretty carefully and pinging the MP team with questions.

Reply:
My notion was that all the inconsequential `CorrectableError`s would be controlled by a single bool. these would be the type that an experienced practitioner walks up to the troublesome input as says, "ah, add autosym there", and walks away unconcerned about the consequences. the more involved "try this, study the output, try that, check the output for red flag" would each be separately controlled. Daniel's rank 1-3 example is in this category.